learningresources/tests/base.py

```diff
@@ -11,14 +11,17 @@
 
 class LoreTestCase(TestCase):
     """Handle often-needed things in tests."""
+    TEST_USERNAME = 'test'
+    TEST_PASSWORD = 'test'
 
     def setUp(self):
         """set up"""
         super(LoreTestCase, self).setUp()
-        self.username = 'test'
-        self.password = 'test'
-        self.user, _ = User.objects.get_or_create(username=self.username)
-        self.user.set_password(self.password)
-        self.user.save()
+        self.user = User.objects.create_user(
+            username=self.TEST_USERNAME, password=self.TEST_PASSWORD
```

Comment:
Sure, that is basically the same case for not allowing empty lists as default arguments, but for this case, I think this is still the right choice.  These are literals, and the person inheriting the class should be able to modify them if they want to override `__init__` and change the user password and name.  I am also not using them as class variables, but object properties here as I don't mind if they are modified, i.e. no `LoreTestCase.TEST_USERNAME` here, I'm using self, so recognizing that they aren't intended to be at the class scope.  I also don't like defining properties outside of `__init__`, so If you have a strong preference I'm happy to move those into the `__init__`, I was just avoiding writing extra code.